.circleci/scripts/parse_commands_to_json.py

```diff
@@ -0,0 +1,75 @@
+from logging import exception
+import os
+from os.path import dirname
+import re
+import json
+
+# Get location of the script
+scriptPath = os.path.dirname(os.path.realpath(__file__))
+# rootPath is root of the directory
+rootPath = dirname(dirname(scriptPath))
+# CommandPath contains linux, mac, windows commands
+commandPath = os.path.join(rootPath, "PowerShell/JumpCloud Commands Gallery")
+pathParts = ['Linux Commands','Mac Commands', 'Windows Commands']
+
+def get_sections(s):
+    for sec in s.split('\n#### '):
+        yield sec if sec.startswith('#### ') else '#### '+sec
+
+cmds = []
+
+for part in pathParts:
+    path = os.path.join(commandPath, part)
+    for file in os.listdir(path):
+        if file.endswith(".md") == False: continue
+
+        filePath = os.path.join(path, file)
+
+        cmd = {
+            'name': '',
+            'type': '',
+            'script': '',
```

Comment:
I realize that after writing this and other discussions, that we should have this match these variables as close as we can to the destination (in this case, very similar to savedAgentCommands in mongo). 

The extra properties are good to go (link, description), but we should change `script` to `command` here.

Reply:
Alright this change is done, related, I added one more automation to this workflow. 

For any script in the repository, the ```setLink.py``` script will take the contents of the Import-JCCommand -URL "someURL" and replace it with a direct link to the file as it will exist in the main branch post-merge.

This addresses two problems I realized I skipped over the weekend:
1) The URLs to the .md files would need to have been updated after we merged to master (this was our current flow)
2) In addition, four of the Git.IO short links we've used for many of these commands were no longer working. Git.IO links are depreciated so I figured it was best to link to files directly.


Any older links like this
```Import-JCCommand -URL 'https://git.io/JJL0x'```
should now be updated to point to their direct path:
```Import-JCCommand -URL "https://github.com/TheJumpCloud/support/blob/master/Linux%20Commands/Linux%20-%20Snap%20-%20Install%20Microsoft%20VS%20Code.md"```